### PR TITLE
Bridge: add kafka bridges

### PR DESCRIPTION
**Pull request by odacremolbap**
Add 2 simple kafka bridges.

Note that:
- there is no target UI.
- these targets are using an unauthenticated kafka.
- production version kafka is not the staging tested version.

### CODE REVIEW

File: bridges/kafka-display/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+## Components
+
+This bridge uses
+
+- Kafka source as the producer of events
+- A broker that receives events from Kafka
+- A trigger that connects the broker with the service
+- An event display service
+
+## Parametrization
```
Comment (sebgoa) [inline bridges/kafka-display/README.md:14]: parameterization

File: bridges/kafka-elasticsearch/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Kafka to Elasticsearch
+
+This sample bridge indexes messages sent throught Kafka into an Elasticsearch cluster.
```
Comment (sebgoa) [inline bridges/kafka-elasticsearch/README.md:3]: through

Comment (sebgoa) [reply bridges/kafka-elasticsearch/README.md:3]: Bridge ?

File: bridges/kafka-display/README.md (hunk 1 above)
Comment (odacremolbap) [reply bridges/kafka-display/README.md:14]: omg, that was a spaniardism

Comment (odacremolbap) [reply bridges/kafka-display/README.md]: https://github.com/triggermesh/bridges/pull/65/commits/ee4c80bc829eda8afee89a0a418450e83fa4be40 expanded the fix to some other occurrences

**Pull request merged**